website/server/libs/payments.js

```diff
@@ -125,13 +130,39 @@ api.addSubToGroupUser = async function addSubToGroupUser (member, group) {
         {name: 'EMAIL', content: getUserInfo(member, ['email']).email},
         {name: 'PAYMENT_METHOD', content: memberPlan.paymentMethod},
         {name: 'PURCHASED_PLAN', content: JSON.stringify(memberPlan)},
-        {name: 'ACTION_NEEDED', content: 'User has joined group plan. Tell them to cancel subscription then give them free sub.'},
+        {name: 'ACTION_NEEDED', content: 'User has joined group plan and has been told to cancel their subscription then email us. Ensure they do that then give them free sub.'},
+        // TODO User won't get email instructions if they've opted out of all emails. See if we can make this email an exception and if not, report here whether they've opted out.
       ]);
     }
 
-    if ((ignorePaymentPlan || ignoreCustomerId) && !customerHasCancelledGroupPlan) return;
+    if ((ignorePaymentPlan || ignoreCustomerId) && !customerHasCancelledGroupPlan) {
+      // member has been added to group plan but their subscription will not be changed
+      // automatically so they need a special message in the email
+      if (memberPlan.paymentMethod === this.constants.GOOGLE_PAYMENT_METHOD) {
+        previousSubscriptionType = 'Google';
```

Comment:
OKay, I see I was mixed up because you added an ||. 
 
I meant to suggest something like this:
```
if (memberPlan.paymentMethod === this.constants.GOOGLE_PAYMENT_METHOD) {
    previousSubscriptionType = this.constants.GOOGLE_PAYMENT_METHOD;
  } else if (memberPlan.paymentMethod === this.constants.IOS_PAYMENT_METHOD) {
    previousSubscriptionType = this.constants.IOS_PAYMENT_METHOD;
  } else if (memberPlan.customerId === this.constants.UNLIMITED_CUSTOMER_ID) {
    previousSubscriptionType = this.constants.UNLIMITED_CUSTOMER_ID';
  } else if (memberPlan.customerId === this.constants.GROUP_PLAN_CUSTOMER_ID) {
    previousSubscriptionType = this.constants.GROUP_PLAN_CUSTOMER_ID';
  } else {
    // this triggers a generic message in the email template in case we forget
    // to update this code for new special cases
    previousSubscriptionType = 'unknown';
  }
```
Since the previousSubscriptionType is the value of the content then it does not affect the constant `PREVIOUS_SUBSCRIPTION_TYPE` (merge tag) on Mandril.

But, it doesn't seem that you are using the previousSubscriptionType for anything other than the email content, so never mind. It is find as is for now. We will probably need to abstract these email stuff to the factory patterns I suggested above later on.